interfaces/builtin/desktop.go

```diff
@@ -463,12 +463,25 @@ dbus (receive, send)
     interface=org.freedesktop.DBus.Properties
     peer=(label=unconfined),
 
-# Allow access to the regular xdg-desktop-portal APIs
+# Allow access and introspection to the regular xdg-desktop-portal APIs
 dbus (send)
     bus=session
     interface=org.freedesktop.portal.*
     path=/org/freedesktop/portal/desktop{,/**}
     peer=(label=unconfined),
+dbus (send)
+    bus=session
+    interface=org.freedesktop.DBus.Properties
+    path=/org/freedesktop/portal/desktop{,/**}
+    peer=(label=unconfined),
+
+# Allow to receive settings change notifications from xdg-desktop-portal
+dbus (receive)
+    bus=session
+    interface=org.freedesktop.portal.Settings
+    path=/org/freedesktop/portal/desktop
+    member=SettingChanged
+    peer=(label=unconfined),
```

Comment:
I think it might be better to synchronise this rule with the plug-side AppArmor rules:

https://github.com/canonical/snapd/blob/18a9ada3b14f521c0487e5a9bab44f4b974e1cb6/interfaces/builtin/desktop.go#L93-L104

There's more signals that just the settings portal `SettingsChanged` signal, so we may as well get them all rather than special casing this one. Similarly, the standard properties interface has a `PropertiesChanged` signal we might want to receive.

Reply:
done